tests/helpers/export_service_utils.py

```diff
@@ -140,3 +141,62 @@ def create_export_message_missing_field_mock(field_to_remove):
     }
     message.pop(field_to_remove)
     return json.dumps(message)
+
+
+def read_csv(csv_content):
+    data = []
+    with open(csv_content, newline="") as csvfile:
+        reader = csv.reader(csvfile)
+        for row in reader:
+            data.append(row)
+    return data
+
+
+def create_export_csv_mock(mocker):
+    header = [
+        "id",
+        "subscription_manager_id",
+        "satellite_id",
+        "display_name",
+        "group_id",
+        "group_name",
+        "os_release",
+        "system_check_in",
+        "state",
+        "tags",
+        "host_type",
+    ]
+
+    row = [
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+        mocker.ANY,
+    ]
```

Comment:
To avoid repetition, and so you don't have to update it each time you update the header:
```suggestion
    row = [mocker.ANY] * len(header)
```

Reply:
Done!

Thanks!!